### ISSUE TITLE
No-code platform app

### ISSUE THREAD

**Issue by chriskapp** [apioo/fusio#360]
If we want to work further on the no-code platform idea, we could try to integrate [blockly](https://developers.google.com/blockly), which is a no-code editor developed by google. Through this we could create an app where a user could build an action without coding. We could create some custom blocks i.e. to get data from table etc.

**Comment by m13v** [comment]
the recurring thing with blockly-style editors is they nail the single action (get data from table, transform, output) but stall the moment a user needs state to flow between actions. building one block feels intuitive; composing ten where block 3 depends on block 1's output is where the visual metaphor starts fighting back and people quietly reach for raw code again. worth scoping your custom blocks around whole flows rather than atomic ops, otherwise you end up reinventing a node graph by accident and inherit all its layout/debug problems.

**Comment by chriskapp** [comment]
@m13v good point and I think also that these UI no-code editors are limited, we also can probably move away from this idea since through our integrated Agent feature a user can now generate actions through an LLM and simple language which then generates the code and this has also not the limit of such an UI no-code editor. So in general we could of course build such an app but I think it is no longer needed.
